README.md

```diff
@@ -76,7 +77,7 @@ Esri welcomes contributions from anyone and everyone. Please see our [guidelines
 
 ## Licensing
 
-Copyright 2023 Esri.
+Copyright 2024 Esri.
```

Comment:
Same question as in Unreal PR. Should we be updating the Copyright?

Reply:
Good question, let me throw that in the group chat.